Error building site using Docker on Apple Silicon Mac

### Description:
I encountered an error while attempting to build the site using Docker on my M1 Mac. The error message is as follows:

```
qemu-x86_64: Could not open '/lib64/ld-linux-x86-64.so.2': No such file or directory
make: *** [container-server] Error 255
```

### Environment:

Operating System: M1 MacOS
Docker Version: 23.0.5

### Steps to Reproduce:

- Install Docker version 23.0.5 on an M1 Mac.
- Execute the [build steps ](https://github.com/kubernetes/contributor-site#using-docker) to generate contents of the site in container.

### Expected Behavior:
The build process should complete successfully without any errors.

### Actual Behavior:
The build process fails with the above error message.

/kind bug

--- a/README.md
+++ b/README.md
@@ -46,6 +46,15 @@ To ensure you can view the site with externally sourced content, run
 `make container-gen-content` before previewing the site by with
 `make container-server`.
 
+**NOTE to Apple Silicon Mac Users**
+
+Before proceeding with the build steps, please ensure that you set the
+`DOCKER_DEFAULT_PLATFORM` environment variable to `linux/amd64` by using the 
+following command: 
+
+```
+export DOCKER_DEFAULT_PLATFORM=linux/amd64
+```
 
 ### Natively
 

